### ISSUE TITLE
newgui installed with setup.py does not start

### ISSUE THREAD

**Issue by reszelaz** [taurus-org/taurus#933]
I have created the a new gui using wizard (taurus newgui) and installed it according to the last pop-up dialog instruction:
![image](https://user-images.githubusercontent.com/6735649/60195874-3cfa3100-983c-11e9-8018-c27b0aaef125.png)
and I'm not able to start the gui using the script (the same pop-up dialog instructs me to do it this way). 
I try it on Windows:

```
Microsoft Windows [Version 10.0.16299.1217]
(c) 2017 Microsoft Corporation. All rights reserved.

C:\Users\IEUser>pip install C:\Users\IEUser\yagui
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Processing c:\users\ieuser\yagui
Requirement already satisfied: taurus[taurus-qt] in c:\python27\lib\site-packages (from tgconf-yagui==0.1.0) (4.5.4a0)
Requirement already satisfied: click in c:\python27\lib\site-packages (from taurus[taurus-qt]->tgconf-yagui==0.1.0) (7.0)
Requirement already satisfied: enum34; python_version < "3.4" in c:\python27\lib\site-packages (from taurus[taurus-qt]->tgconf-yagui==0.1.0) (1.0.4)
Requirement already satisfied: numpy>=1.1 in c:\python27\lib\site-packages (from taurus[taurus-qt]->tgconf-yagui==0.1.0) (1.14.1)
Requirement already satisfied: pint>=0.8 in c:\python27\lib\site-packages (from taurus[taurus-qt]->tgconf-yagui==0.1.0) (0.8.1)
Requirement already satisfied: future in c:\python27\lib\site-packages (from taurus[taurus-qt]->tgconf-yagui==0.1.0) (0.17.1)
Requirement already satisfied: guiqwt>=3; extra == "taurus-qt" in c:\python27\lib\site-packages (from taurus[taurus-qt]->tgconf-yagui==0.1.0) (3.0.3)
Requirement already satisfied: lxml>=2.1; extra == "taurus-qt" in c:\python27\lib\site-packages (from taurus[taurus-qt]->tgconf-yagui==0.1.0) (3.4.4)
Requirement already satisfied: ply>=2.3; extra == "taurus-qt" in c:\python27\lib\site-packages (from taurus[taurus-qt]->tgconf-yagui==0.1.0) (3.6)
Requirement already satisfied: Pillow in c:\python27\lib\site-packages (from guiqwt>=3; extra == "taurus-qt"->taurus[taurus-qt]->tgconf-yagui==0.1.0) (5.0.0)
Requirement already satisfied: guidata>=1.7.0 in c:\python27\lib\site-packages (from guiqwt>=3; extra == "taurus-qt"->taurus[taurus-qt]->tgconf-yagui==0.1.0) (1.7.6)
Requirement already satisfied: SciPy>=0.7 in c:\python27\lib\site-packages (from guiqwt>=3; extra == "taurus-qt"->taurus[taurus-qt]->tgconf-yagui==0.1.0) (1.0.0)
Requirement already satisfied: PythonQwt>=0.5.0 in c:\python27\lib\site-packages (from guiqwt>=3; extra == "taurus-qt"->taurus[taurus-qt]->tgconf-yagui==0.1.0) (0.5.5)
Building wheels for collected packages: tgconf-yagui
  Building wheel for tgconf-yagui (setup.py) ... done
  Stored in directory: c:\users\ieuser\appdata\local\temp\pip-ephem-wheel-cache-yi5ty0\wheels\7a\6c\ad\37d5206e0153d1fa356557a78556a2e84bcb74935f7878c745
Successfully built tgconf-yagui
Installing collected packages: tgconf-yagui
Successfully installed tgconf-yagui-0.1.0

C:\Users\IEUser>yagui

C:\Users\IEUser>
```
The application never showsup and the prompt is returned immediatelly.

Note that the gui starts correctly using: `taurus gui tgconf_yagui`

**Comment by stanislaw55** [comment]
This problem may occur on Windows due to usage of `console_scripts` and not `gui_scripts` in entrypoint. Please refer to cookiecutter project in taurus-org. 

Please check if changing to `gui_scripts` before you install gui resolves problem.

**Comment by reszelaz** [comment]
Thanks @stanislaw55 for your help!

The setup.py which was generated in my case already uses the gui_scripts:

```python
"""
This file was autogenerated by taurusgui --new-gui.
"""

from setuptools import setup, find_packages

# version (bumpversion style)
__version = '0.1.0'

# Example of classifier
classifiers = [
    'Development Status :: 3 - Alpha',
    'Intended Audience :: End Users/Desktop',
    'Topic :: Scientific/Engineering',
    'Environment :: X11 Applications :: Qt',
    'Programming Language :: Python :: 2.7',
    'Programming Language :: Python :: 3.5'
]

_entry_points = {
    "gui_scripts": ["yagui=tgconf_yagui:run"]
}

setup(
    name="tgconf_yagui",
    version=__version,
    include_package_data=True,
    packages=find_packages(),
    entry_points=_entry_points,
    url='http://www.taurus-scada.org/en/latest/',
    keywords='APP',
    description='yagui Taurus GUI',
    requires=['setuptools (>=1.1)'],
    install_requires=['taurus[taurus-qt]'],
    classifiers=classifiers
)
```

**Comment by stanislaw55** [comment]
In that case, I'm not sure what can cause such behaviour.
I've realized that `taurus newgui` uses internal template instead of `cookiecutter-tarusupackage`.

**Comment by cmft** [comment]
I will say that the problem is in the init template of the taurusgui. The entry point refers to a method that has been changed. I will test it and create a PR.

**Comment by cpascual** [comment]
> I've realized that taurus newgui uses internal template instead of cookiecutter-tarusupackage.

Yes, my idea was to start using cookiecutter and replace the old template., but it is still TODO

**Comment by cpascual** [comment]
A quick fix is proposed in #935. Please try it.
